doc/dev/release.md

```diff
@@ -274,6 +274,11 @@ To github.com:operator-framework/operator-sdk.git
 
 Now that the branch exists, you need to make the post-release PR for the new release branch. To do this, simply follow the same steps as in [step 3](#3-create-a-pr-for-post-release-version-and-changelogmd-updates) with the addition of changing the branch name in the `gopkgtoml` scaffold from `master` to the new branch (for example, `v1.3.x`). Then, make the PR against the new branch.
 
+### 6. Updating Homebrew formula
+
+We support installing via [Homebrew][homebrew], so we need to update the operator-sdk [Homebrew formula][homebrew-formula] once the release is cut. Follow the instructions in the Homebrew contribution [README][homebrew-readme] and open a PR against the [repository][homebrew-repo].
```

Comment:
Not sure if it's helpful to include it, but [this blog post](https://thecoatlessprofessor.com/programming/updating-a-homebrew-formula/) helped me when I was doing the update. WDYT?

Reply:
Do you think it would be easier if we just added a script in our repo that does [this](https://github.com/operator-framework/operator-sdk/pull/1224#issuecomment-475816727) inside the [`linuxbrew/brew`](https://hub.docker.com/r/linuxbrew/brew) docker container?

Reply:
That didn't work for me, for some reason on my mac. I had to follow the more manual steps of doing `brew edit operator-sdk`, replace the url and `shasum -a 256 <operator-sdk-binary>` to get the sha. But can try doing in the docker container.

On a somewhat related note, we should really include the SHA sum in our releases, wdyt? As per https://github.com/prometheus/prometheus/releases

Reply:
So I tried doing the above command, for this we need git credentials, so I managed to only got it to work by the following steps:
```
docker run -t -d linuxbrew/brew:latest
docker exec -it <CONTAINER_ID> /bin/bash`
git config --global github.name <GITHUB-USERNAME>
git config --global github.token <GITHUB-TOKEN>
brew bump-formula-pr --strict --url=https://github.com/operator-framework/operator-sdk/archive/v0.7.0.tar.gz --sha256="some-sha" operator-sdk
```
(I couldn't ofcourse test the whole process as there is nothing to bump, but it seems to give me a warning I can expect)

When I tried to more automate this, by doing the following it doesn't work:
```
docker run linuxbrew/brew:latest /bin/bash -c 'git config --global github.name <GITHUB-USERNAME> && git config --global github.token <GITHUB-TOKEN> &&
brew bump-formula-pr --strict --url=https://github.com/operator-framework/operator-sdk/archive/v0.7.0.tar.gz --sha256="some-sha" operator-sdk
```

I get the following error, I checked and git name and token is set: 🤷‍♀️ 
```
Error: GitHub You must be logged in to do that.:The GitHub credentials in the macOS keychain may be invalid.
```

Any suggestions? Otherwise for now I would just document the steps that worked for me e.g. exec into the container and run the steps manually. (I also tried setting the credantials in a Dockerfile, building that and running, got the same error)

Reply:
@joelanford ^ in case you missed this :)

Reply:
@LiliC thanks for the poke! Sounds reasonable to just document the steps for now.

Reply:
@joelanford Done, hopefully, the steps make sense. I didn't create the PR using that command, so we need to test it next to make sure it works, but there are the backup docs in case something goes wrong. :) PTAL.